drainer/translator/table_info.go

```diff
@@ -19,5 +19,7 @@ import "github.com/pingcap/parser/model"
 type TableInfoGetter interface {
 	TableByID(id int64) (info *model.TableInfo, ok bool)
 	SchemaAndTableName(id int64) (string, string, bool)
-	IsDroppingColumn(id int64) bool
+	CanAppendDefaultValue(id int64, schemaVersion int64) bool
+	// IsDroppingColumn(id int64) bool
+	TableBySchemaVersion(version int64, tid int64) (info *model.TableInfo, ok bool)
```

Comment:
The variable name here is a bit confusing.
- `id` and `tid` are both table's id.
- `version` and `schemaVersion` are both schema's version.
- the order of id and version is not same.